backend/apollo/elections/models/election.py

```diff
@@ -44,6 +47,17 @@ def can_be_opened(self) -> bool:
             and not self.questions.filter(answers__isnull=True).exists()
         )
 
+    def _open_votes(self):
+        for vote in Vote.objects.filter(question__election=self):
+            try:
+                decrypted_answer_id = decrypt(self.secret_key, vote.answer_ciphertext)
+                answer = Answer.objects.get(pk=decrypted_answer_id)
+                if answer.question.election == self:
+                    vote.answer = answer
+                    vote.save()
+            except (CryptoError, Answer.DoesNotExist):
```

Comment:
Are we going to have any mechanism to verify (with zero knowledge) that a vote is indeed for one of the election's questions?

Also, I'd move the `if` to `try except else`.

Reply:
We have to discuss how to do it first.

Good idea, I moved it